src/commands/dev/stop.ts

```diff
@@ -96,8 +97,8 @@ export default class DevStop extends BaseCommand {
       });
     } else {
       // If there's no socket at all, dev is running in detached mode and we can stop it with docker compose without
-      // working about the health check restarting anything.
-      this.runComposeStop(name);
```

Comment:
I think I purposely didn't await this and i'm trying to now remember why...

Reply:
Ah - right - I think that when `await`-ing this, we never actually do the whole ``Waiting for ${name} to stop...``  below this and `waitForEnviromentToStop` is what we actually want to block on

Reply:
I see what you mean. I just moved that line to provide the same experience